codestyle/checkstyle.xml

```diff
@@ -71,5 +71,26 @@
       <property name="illegalPattern" value="true"/>
       <property name="message" value="Use Comparators.naturalNullsFirst() instead of Ordering.natural().nullsFirst()"/>
     </module>
+
+    <module name="Regexp">
```

Comment:
I think yes, because they are 1) never needed in the codebase so far 2) bug-prone, people use them instead of Infinity constants.